ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

```diff
@@ -173,6 +174,33 @@ platform:
     username: "${GOVC_USERNAME}"
     ${RESOURCE_POOL_DEF}
 EOF
+else
+  cat >>"${CONFIG}" <<EOF
+baseDomain: $base_domain
+$MACHINE_POOL_OVERRIDES
+platform:
+  vsphere:
+    vcenters:
+    - datacenters:
+       - ${vsphere_datacenter}
+      password: ${GOVC_PASSWORD}
+      port: 443
+      server: ${vsphere_url}
+      user: ${GOVC_USERNAME}
+    failureDomains:
+    - name: generated-failure-domain
+      region: generated-region
+      server: ${vsphere_url}
+      topology:
+        computeCluster: /${vsphere_datacenter}/host/${vsphere_cluster}
+        datacenter: ${vsphere_datacenter}
+        datastore: /${vsphere_datacenter}/datastore/${vsphere_datastore}
+        networks:
+        - ${vsphere_portgroup}
+        resourcePool: /${vsphere_datacenter}/host/${vsphere_cluster}//Resources
```

Comment:
Here we are supposed to use `${RESOURCE_POOL_DEF}` ?

Reply:
I think so, here's the install config generated by default
```
    vcenters:
    - datacenters:
       - DEVQEdatacenter
      password: xxx
      port: 443
      server: vcenter.devqe.ibmc.devcluster.openshift.com
      user: xxx@devqe.ibmc.devcluster.openshift.com

    failureDomains:
    - name: generated-failure-domain
      region: generated-region
      server: vcenter.devqe.ibmc.devcluster.openshift.com
      topology:
        computeCluster: /DEVQEdatacenter/host/DEVQEcluster
        datacenter: DEVQEdatacenter
        datastore: /DEVQEdatacenter/datastore/vsanDatastore
        networks:
        - devqe-segment-221
        resourcePool: /DEVQEdatacenter/host/DEVQEcluster//Resources
      zone: generated-zone
```

Reply:
I meant in this script, there is a dedicated var - `RESOURCE_POOL_DEF` to customize the placement for the cluster. So I think here we should use the existing var here.

Reply:
Good catch, I think we can also use RESOURCE_POOL_DEF here